plugins/event_notification/providers/email/lib/EmailNotificationTemplateConfiguration.php

```diff
@@ -90,15 +90,27 @@ protected function getHeaderField( $headerName , $properties)
 					$name = new Kaltura_Client_Type_StringValue();
 					$name->value = $properties[$headerNameProperty];
 				}
-
-				$recipient = new Kaltura_Client_EmailNotification_Type_EmailNotificationRecipient();
-				$recipient->email = $email;
-				$recipient->name = $name;
-
-				$recipientProvider = new Kaltura_Client_EmailNotification_Type_EmailNotificationStaticRecipientProvider();
-				$recipientProvider->emailRecipients = array();
-				$recipientProvider->emailRecipients[] = $recipient;
-
+				$delimiter = ";";
```

Comment:
I want to be able to handle both in the same string so we'll catch also this
a@aol.com;b@aol.com,c@aol.com
You can either use preg_split or just replace all "," with ";" and use the one delimiter